packages/render-fidelity-tools/src/workflows/update-screenshots.ts

```diff
@@ -39,12 +39,28 @@ const goldensDirectory = join(rootDirectory, 'goldens');
 const renderersDirectory = join(rootDirectory, 'renderers')
 
 let scenarioWhitelist: Set<string>|null = null;
+let rendererWhitelist: Set<string>|null = null;
+const rendererList = new Set(config.renderers.map((renderer: any) => {
```

Comment:
I feel like this line is a little confusing. If you have more clear and easy to understand way to covert an array's properties to a set, im happy to learn.